keras2onnx/_builtin.py

```diff
@@ -2360,10 +2380,14 @@ def convert_tf_variable_v2(scope, operator, container):
 def convert_tf_where(scope, operator, container):
     oopb = OnnxOperatorBuilder(container, scope)
     node = operator.raw_operator
+    if container.target_opset < 13:
+        op_version = 9
+    else:
+        op_version = 13
```

Comment:
it looks there is no difference on using opset 11 or opset 13. Conventionally,  the converted model will use the minimal opset number, instead of the opset specified by the user.

Reply:
synced offline.